src/crypto/BUILD.gn

```diff
@@ -105,7 +114,11 @@ if (chip_crypto == "openssl") {
   import("//build_overrides/mbedtls.gni")
 
   source_set("cryptopal_mbedtls") {
-    sources = [ "CHIPCryptoPALmbedTLS.cpp" ]
+    sources = [
+      "CHIPCryptoPALmbedTLS.cpp",
+      "RawKeySessionKeystore.cpp",
+      "RawKeySessionKeystore.h",
```

Comment:
Why not *always* include RawSessionKeystore to every target, and just rely on the fact that if you use a different one, that generic RawSessionKeystore (that should always work in theory) would just never be linked?

Reply:
For PSA, this implementation is just wrong, but I can add if on `chip_crypto != "psa"`.